test/api/connectors/MockHttpClient.scala

```diff
@@ -43,47 +43,50 @@ trait MockHttpClient extends MockFactory {
            _: Seq[(String, String)],
            _: HttpReads[T],
            hc: HeaderCarrier,
-           _: ExecutionContext) => {
-            actualUrl shouldBe url
-            actualParams shouldBe parameters
-
-            val headersForUrl = hc.headersForUrl(config)(actualUrl)
-            assertHeaders(headersForUrl, requiredHeaders, excludedHeaders)
-          }
+           _: ExecutionContext) =>
+            {
+              actualUrl shouldBe url
+              actualParams shouldBe parameters
+
+              val headersForUrl = hc.headersForUrl(config)(actualUrl)
+              assertHeaders(headersForUrl, requiredHeaders, excludedHeaders)
+            }
         })
     }
 
     def post[I, T](url: String,
                    config: HeaderCarrier.Config,
                    body: I,
-                   requiredHeaders: Seq[(String, String)] = Seq.empty,
-                   excludedHeaders: Seq[(String, String)] = Seq.empty): CallHandler[Future[T]] = {
+                   requiredHeaders: Seq[(String, String)] = Nil,
```

Comment:
Just out of curiosity - why the preference of Nil over Seq.empty?

Reply:
It's shorter